release-team/role-handbooks/emeritus-adviser/README.md

```diff
@@ -38,7 +38,14 @@ Due to timing (see below)
 
 Weeks: -2 to 0
 
-At least 2 weeks before the new release cycle begins, the EA is responsible for posting the "Assemble the X.Y Release Team" issue in the SIG-Release repo ([example](https://github.com/kubernetes/sig-release/issues/776)).  They should then make sure that the new Release Lead is officially nominated and chosen, and then issue a call for nominations/volunteers for the other Role Leads.
+At least 2 weeks before the new release cycle begins, the EA is responsible for
+posting the "Assemble the X.Y Release Team" issue in the SIG-Release repo
+([example](https://github.com/kubernetes/sig-release/issues/776)). Before doing
+that, it's required to reach consensus with all [SIG Release Leads](https://github.com/orgs/kubernetes/teams/sig-release-leads)
+about the next Release Team Lead as well as EA, which can be done by using a
+private Slack group (or a mail thread). If consensus has been reached, then the
```

Comment:
```suggestion
Slack group DM or an email thread to sig-release-leads@kubernetes.io. If consensus has been reached, then the
```

Two fixes:
- let's be careful about using the word "private" w.r.t. Slack comms, given past situations
- naming the SIG Release Leads mailing list